controlplane/kubeadm/internal/controllers/controller.go

```diff
@@ -255,6 +255,12 @@ func (r *KubeadmControlPlaneReconciler) reconcile(ctx context.Context, cluster *
 		return ctrl.Result{}, err
 	}
 
```

Comment:
Should we drop l.153-156?

Reply:
Woops, looks like i forgot to delete those lines. Will do.